examples/rust/BUILD

```diff
@@ -0,0 +1,13 @@
+# Public notice: this file is for internal documentation, testing, and
+# reference only. Note that repo maintainers can freely change any part of the
+# repository code at any time.
+package(default_visibility = ["//visibility:public"])
+
+# Rust
+load("@io_bazel_rules_docker//rust:image.bzl", "rust_image")
```

Comment:
You need to reference [these from the WORKSPACE](https://github.com/bazelbuild/rules_rust/blob/master/README.md#setup)